<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CModelMappingForm</class>
 <widget class="QFrame" name="CModelMappingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model editor</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="vl_ModelMappingForm">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_ModelMappingForm">
     <property name="title">
      <string>Simulator model</string>
     </property>
     <layout class="QGridLayout" name="gl_ModelMappingForm">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="BlackGui::CLabelAndIcon" name="lai_Id">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="2">
       <widget class="BlackGui::Components::CSimulatorSelector" name="selector_Simulator">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="pb_Stash">
        <property name="text">
         <string>Stash</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lbl_Description">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="buddy">
         <cstring>le_Description</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="le_Description">
        <property name="placeholderText">
         <string>Model description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="lbl_Updated">
        <property name="text">
         <string>Last updated:</string>
        </property>
        <property name="buddy">
         <cstring>le_LastUpdated</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lbl_ModelKey">
        <property name="text">
         <string>Model key:</string>
        </property>
        <property name="buddy">
         <cstring>le_ModelKey</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="le_LastUpdated"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_Id">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Id</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_ModelKey">
        <property name="placeholderText">
         <string>Model  key</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_Name">
        <property name="placeholderText">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_Name">
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="buddy">
         <cstring>le_Name</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="BlackGui::Editors::CValidationIndicator" name="val_Indicator">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="BlackGui::Components::CModelModeSelector" name="selector_ModelMode">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CModelModeSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/components/modelmodeselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::CLabelAndIcon</class>
   <extends>QFrame</extends>
   <header>blackgui/labelandicon.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CValidationIndicator</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/validationindicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Id</tabstop>
  <tabstop>le_ModelKey</tabstop>
  <tabstop>le_Name</tabstop>
  <tabstop>le_Description</tabstop>
  <tabstop>pb_Stash</tabstop>
  <tabstop>le_LastUpdated</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
